includes/polling/entity-physical/state.inc.php

```diff
@@ -11,13 +11,26 @@
                 $entity['entPhysicalIndex'],
                 $entity['subindex'],
                 $entity['group'],
-                $entity['key'],
+                $entity['key']
             )
         );
     } else {
         if ($entPhysical_state[$entity['entPhysicalIndex']][$entity['subindex']][$entity['group']][$entity['key']] != $entity['value']) {
-            echo 'no match!';
-        }
+            echo 'no match!' . ' ' . $entity['entPhysicalIndex'] . ' ' . $entity['subindex'] . ' ' . $entity['key'] . ': ' . $entPhysical_state[$entity['entPhysicalIndex']][$entity['subindex']][$entity['group']][$entity['key']] . ' ' . $entity['value'] . "\n";
+            dbUpdate(
+		array('value' => $entPhysical_state[$entity['entPhysicalIndex']][$entity['subindex']][$entity['group']][$entity['key']]),
```

Comment:
This value is accessed 3 times and is rather long, a temporary variable is probably in order.